src/utils/validations.js

```diff
@@ -20,6 +20,28 @@ export const validateTicker = (value) => {
   return isValid ? undefined : VALIDATION_MESSAGES.TICKER
 }
 
+export const validateWhitelistMin = ({ min, max, decimals }) => {
+  const listOfErrors = composeValidators(
+    isRequired(),
+    isNonNegative(),
+    isDecimalPlacesNotGreaterThan(`Decimals should not exceed ${decimals} places`)(decimals),
+    isLessOrEqualThan('Should be less or equal than max')(max)
+  )(min)
+
+  return listOfErrors ? listOfErrors.shift() : ''
```

Comment:
Shouldn't this return `undefined` when there are no errors?

Reply:
yes, fixed